packages/frontend/preview/src/lib/map/map_preview.svelte

```
@@ -0,0 +1,125 @@
+    height = 480,
+    showCollision = false,
+    showZBands = false,
+    zoom = 1,
+  }: Props = $props();
+
+  let canvasEl: HTMLCanvasElement | undefined = $state(undefined);
+  let error = $state<string | undefined>(undefined);
+  let loaded = $state(false);
+
+  onMount(async () => {
+    if (!canvasEl) return;
```

Biome (2.5.7)

[error] Block statements are preferred in this position.

(lint/style/useBlockStatements)